src/api/axiosInstance.js

```diff
@@ -41,6 +41,13 @@ const createRequestInstance = (resolve, timeout) => {
   instance.interceptors.response.use((response) => {
     return response;
   }, (error) => {
+    // Update error message in redux depending on the status code
+    const statusCode = error.response.status;
+    if (statusCode === 503) {
+      store.dispatch(updateRequestErrorMessage('Our server has reached its requests limit. You should wait a few seconds and try again.'));
```

Comment:
I prefer: "Rate limit exceeded. Sorry about that. You should wait a few seconds and try again."
What do you think?

Reply:
I think it's better too, changed it!